modules/token/lib/token-apr-handler/sources/reaper.ts

```diff
@@ -0,0 +1,50 @@
+import { abi } from './abis/reaperStrategy'
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { MulticallWrapper } from "ethers-multicall-provider";
+import { Contract } from "ethers";
+
+
+const jsonRpcProvider = new JsonRpcProvider('https://arb1.arbitrum.io/rpc', 42161 /*ARBITRUM*/)
+const provider = MulticallWrapper.wrap(jsonRpcProvider)
```

Comment:
why is this multicall wrapper needed?

Reply:
I used it so calls performed simultaneously can be automatically batched. Like in this case:
`const getAprs = () => {

  const calls = Object.keys(strategiesMap).map(async (coin) => {
    const contract = new Contract(strategiesMap[coin], abi, provider)
    return contract.averageAPRAcrossLastNHarvests(3)
  })

  return Promise.all(calls)
}`